### ISSUE TITLE
NoMethodError: undefined method `schema_format' for ActiveRecord::Base:Class (NoMethodError)

### ISSUE THREAD

**Issue by colszowka** [jenseng/hair_trigger#117]
Hey all :wave: 

After upgrading our Rails (7.12 on Ruby 3.2.2) app to the latest hairtrigger version 1.1.0, when running `rake db:migrate` we get the following exception:

```
NoMethodError: undefined method `schema_format' for ActiveRecord::Base:Class (NoMethodError)
```

We are using postgres and the sql dump format. Downgrading the gem back to 1.0.0 resolves the issue for our application.

**Comment by ollym** [comment]
Fixed https://github.com/jenseng/hair_trigger/pull/118

**Comment by jenseng** [comment]
thanks everyone, and sorry for the trouble! the fix has been released in v1.1.1

**Comment by colszowka** [comment]
Wow that was a quick turnaround, thanks @ollym @jenseng!! I checked on our app and on 1.1.1 the issue is resolved :+1:
